test/images/image-util.sh

```diff
@@ -117,23 +141,46 @@ push() {
   docker_version_check
   TAG=$(<"${IMAGE}"/VERSION)
   if [[ -f ${IMAGE}/BASEIMAGE ]]; then
-    archs=$(listArchs)
+    base_images=$(listBaseImages)
+    # NOTE(claudiub): if the REMOTE_DOCKER_URL var is not set, or it is an empty string, we must skip
+    # pushing the Windows image and including it into the manifest list.
+    if [[ ! -v "REMOTE_DOCKER_URL" || -z "${REMOTE_DOCKER_URL}" ]]; then
+      if [[ ! -z "$(printf "%s\n" $base_images | grep '^windows')" ]]; then
+        echo "Skipping pushing the image '${IMAGE}' for Windows. REMOTE_DOCKER_URL should be set, containing the URL to a Windows docker daemon."
+        base_images=$(printf "%s\n" $base_images | grep -v "^windows")
+      fi
+    fi
   else
-    archs=${!QEMUARCHS[@]}
+    # prepend linux/ to the QEMUARCHS items.
+    base_images=`printf 'linux/%s\n' "${!QEMUARCHS[@]}"`
```

Comment:
Not sure why you chose backticks here over `$(...)`

Reply:
Personal preference, honestly, but I do try to keep it consistent with how the file is written. Seems like I forgot about this.